[CUBRIDMAN-185] Added vacuumdb --dump output (#441)
CUBRID/cubrid-manual: en/admin/config.rst

    **regexp_engine** is a parameter to choose a library in which regular expression operators and functions will perform. **cppstd** or **re2** can be set and the default value is **re2**. For more information on regular expression functionalities, refer to :doc:`/sql/function/regex_fn`.

**vacuum_ovfp_check_threshold**

>  **vacuum_ovfp_check_threshold** collects information as a management target when the number of overflow pages of a terminal node that must be read to process one key in BTREE is greater than the specified value. The default is 1000.

Reviewer: ```suggestion
 **vacuum_ovfp_check_threshold** collects index information when the number of leaf's overflow pages to be read is greater than the value, when index vacuum is performed. The default is 1000 pages.
```

**vacuum_ovfp_check_duration**

>  **vacuum_ovfp_check_duration** sets the period to keep the information collected by the **vacuum_ovfp_check_threshold** setting. Information that has not been newly updated within the set period is deleted. The unit of the input value is seconds, and the default value is 2678400, which corresponds to 31 days.

Reviewer: ```suggestion
 **vacuum_ovfp_check_duration** sets the period to keep the information collected by the **vacuum_ovfp_check_threshold**. Information that has not been updated within the setting period is deleted. The unit of the input value is seconds, and the default value is 2678400 seconds, which corresponds to 31 days.
```
